tempesta_fw/t/functional/helpers/prepare.py

```diff
@@ -0,0 +1,13 @@
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+from . import remote
+
+def configure_tcp():
+    """ configuring TCP for faster reuse the same TCP ports """
```

Comment:
Need to comment why the long reuse is the issue. It'll be hard to remember in a few weeks.

Also `configure_tcp()` is just one single step in whole preparing procedure, and #857 is also required. Please, add high level prepare function and add TODO for #857

Reply:
No need to write TODO. I will add this checks when will do #857